runtime/drivers/duckdb/connectors.go

```diff
@@ -118,6 +121,86 @@ func (c *connection) ingest(ctx context.Context, env *connectors.Env, source *co
 	return summary, nil
 }
 
+func (c *connection) ingestMotherduckData(ctx context.Context, source *connectors.Source, env *connectors.Env) (*drivers.IngestionSummary, error) {
+	err := c.WithConnection(ctx, 1, func(ctx, ensuredCtx context.Context) error {
+		res, err := c.Execute(ctx, &drivers.Statement{Query: "SELECT current_database();"})
+		if err != nil {
+			return c.checkErr(err)
+		}
+		defer res.Close()
+
+		res.Next()
+		var localDB string
+		if err := res.Scan(&localDB); err != nil {
+			return c.checkErr(err)
+		}
+
+		// get token
+		token := env.Variables["TOKEN"]
```

Comment:
Is this lookup case insensitive?

Reply:
In a sense yes. Before setting this we convert every variable to uppercase.